run/events-storage/anthos.md

```diff
@@ -0,0 +1,96 @@
+# Events for Cloud Run on Anthos – GCS tutorial
+
+This sample shows how to create a service that processes GCS events.
+
+## Setup
+
+Login to gcloud:
+
+```sh
+gcloud auth login
+```
+
+Configure project id:
+
+```sh
+gcloud config set project [PROJECT-ID]
+```
+
+Configure environment variables:
+
+```sh
+MY_RUN_SERVICE=gcs-service
+MY_RUN_CONTAINER=gcs-container
+MY_GCS_TRIGGER=gcs-trigger
+MY_GCS_BUCKET=gcs-bucket-$(gcloud config get-value project)
+```
+
+## Quickstart
+
+Set cluster name, location and platform:
+
+```sh
+gcloud config set run/cluster events-cluster
+gcloud config set run/cluster_location europe-west1-b
```

Comment:
Should this be eu? Should we give an option to use other locations?

Reply:
Ran into this error since my cluster was in the US: 

```Could not find [events-cluster] in [europe-west1-b].
Did you mean [events-cluster] in [us-central1-c]?```

Reply:
It can be EU or US..This is meant to be just an example of a cluster name and location. We can externalize them into env vars, if you think it's better that way.

Reply:
I think it would be better to put it into env vars given the creation of the clusters is outside this tutorial and could easily be in another region.

Reply:
I commited a change to externalize these into env variables